### PR TITLE
Que 335 elm formatted text fails parsing bold

### PR DESCRIPTION
**Pull request by luizcarlos1405**
Fixes https://linear.app/noredink/issue/QUE-335/elm-formatted-text-fails-parsing-bold-sometimes.

### CODE REVIEW

**Comment by juanedi** [comment]
Hello there! Sorry, I didn't get to review this today and I'll be OOO on Monday. Feel free to request another reviewer if you'd rather have this merged soon.

File: src/FormattedText.elm [hunk 1]
```diff
@@ -398,7 +398,10 @@ slice =
 -}
 lines : FormattedText markup -> List (FormattedText markup)
 lines formatted =
-    split "\n" formatted
+    formatted
+        |> split "\n\r"
```
Comment (stoeffel) [inline src/FormattedText.elm]: We should use `String.lines` which breaks on any linebreak characters https://github.com/elm/core/blob/8e99db3fcbb101838c36d534a7be7add6b6d739e/src/Elm/Kernel/String.js#L177
Additionally, we should mention changes like this in the changelog.

Comment (stoeffel) [reply src/FormattedText.elm]: oh nevermind this uses split on a formatted text.

Comment (stoeffel) [reply src/FormattedText.elm]: how about we do it with the regex like for `words`? that seems a little more performant.

**Review by stoeffel — CHANGES_REQUESTED** [review]
Looks good. I only have one change request. Thanks for porting the tests!

**Review by stoeffel — APPROVED** [review]
Addressed my own comments.

**Pull request merged**